system-int-test/test/cmr/system_int_test/admin/cache_api_test.clj

```diff
@@ -71,18 +71,49 @@
     (testing "list caches"
       (are [url caches]
         (is (= (set caches) (set (list-caches-for-app url admin-read-token))))
-
-        (url/indexer-read-caches-url) ["acls" "indexer-index-set-cache" "token-imp" "kms" "health"
-                                       "collection-granule-aggregation-cache" "humanizer-cache"
-                                       "usage-metrics-cache"]
-        (url/index-set-read-caches-url) ["token-imp" "health"]
-        (url/mdb-read-caches-url) ["token-imp" "health"]
-        (url/ingest-read-caches-url) ["token-imp" "providers" "acls" "token-user-ids" "kms"
-                                      "health" "humanizer-alias-cache" "write-enabled"]
-        (url/access-control-read-caches-url) ["acls" "health" "group-ids-guids" "write-enabled"]
-        (url/search-read-caches-url) ["acls" "collections-for-gran-acls" "has-granules-map"
-                                      "index-names" "token-imp" "token-sid" "token-user-id" "health"
-                                      "xsl-transformer-templates" "metadata-cache" "kms" "write-enabled"])
+        (url/indexer-read-caches-url) [
```

Comment:
Don't bother changing this, but it occurs to me that it would be better if we used the respective vars here instead of string values. That way we can change cache names if needed without updating tests.